chore(website): Update docs and front page (#6854)
cloudquery/cloudquery: website/pages/docs/index.mdx

<Features page="docs" detailed={false} />

CloudQuery is an [open-source](https://github.com/cloudquery/cloudquery),

> high-performance data integration built for developers.

Reviewer: ```suggestion
high-performance data integration framework built for developers.
```

Reviewer: Or platform? Either way, seems like there's a word missing here

Author: yeah platform seems right.

CloudQuery extracts, transforms, and loads configuration from cloud APIs to
variety of supported destinations such as databases, data lakes, or streaming platforms
for further analysis.

## Why CloudQuery?

> - **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.

Reviewer: ```suggestion
- **Open source**: Extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
```

On the home page we used "extensible". Similar meanings, but let's keep it consistent

> - **Blazing fast**: CloudQuery concurrency system utilizes the excellent Go concurrency model with light-weight goroutines.

Reviewer: ```suggestion
- **Blazing fast**: CloudQuery is optimized for performance, utilizing the excellent Go concurrency model with light-weight goroutines.
```

> - **Deploy anywhere**: CloudQuery plugins are single binary executables and can be deployed and run anywhere.

Reviewer: ```suggestion
- **Deploy anywhere**: CloudQuery plugins are single-binary executables and can be deployed and run anywhere.
```

> - **Pre-built queries**: CloudQuery maintains number of out-of-the-box security and compliance policies for cloud infrastructure.

Reviewer: ```suggestion
- **Pre-built queries**: CloudQuery maintains out-of-the-box security and compliance policies for cloud infrastructure.
```

> - **Eliminate data silos**: Eliminate data silos across your organization, security, infrastructure, marketing and finance teams.

Reviewer: ```suggestion
- **Eliminate data silos**: Eliminate data silos across your organization, unifying data between security, infrastructure, marketing and finance teams.
```

> - **Unlimited scale**: CloudQuery plugins are stateless and can scaled easily horizontally on any executor such as k8s, batch jobs.

Reviewer: ```suggestion
- **Unlimited scale**: CloudQuery plugins are stateless and can be scaled horizontally on any platform, such as VMs, Kubernetes or batch jobs.
```
